projects/extension/src/content/ExtensionMessageRouter.ts

```diff
@@ -60,7 +60,7 @@ export class ExtensionMessageRouter {
       debug(`RECEIVED MESSAGE FROM ${chainId} PORT`, data)
       sendMessage({
         type,
-        payload,
+        jsonRpcMessage: payload,
```

Comment:
There are 2 issues here:
1) Not all the messages received from the background are going to be `rpc` messages
2) there is no such thing as a `payload` anymore, so that will always be `undefined`